### ISSUE TITLE
ARIA labels in navigation bar

### ISSUE THREAD

**Issue by obr42** [beer-garden/react-ui#360]
- The two menu groups (navigation and version/docs) do not have aria-labels
  - aria-haspopup and aria-expanded missing on Admin and Options parent menuitems
  - menuitems need aria-labels - Admin submenu items should have `Admin` in the aria-label
